documents/knowledge/yaml_to_txt.py

```diff
@@ -0,0 +1,37 @@
+import os
```

Comment:
Do we really need a txt?

Reply:
i think it's good to have an autogenerated txt for documentation. we can probably gitignore the generated txt anyway